src/openms/source/CHEMISTRY/NucleicAcidSpectrumGenerator.cpp

```diff
@@ -66,10 +66,10 @@ namespace OpenMS
     defaults_.setValue("add_c_ions", "false", "Add peaks of c-ions to the spectrum");
     defaults_.setValidStrings("add_c_ions", {"true","false"});
 
-    defaults_.setValue("add_d_ions", "false", "Add peaks of d-ions to the spectrum");
+    defaults_.setValue("add_d_ions", "false", "Add peaks of d-ions to the spectrum (nucleotide sequences only)");
```

Comment:
is this really needed here? I would assume that the options in this class are only relevant to NAs anyway

Reply:
probably not honestly, just added it because the a-B ions already had that reminder

Reply:
hmm it seems to me that this should probably just be a comment and not some description in the parameter class.
Would you mind just removing it while you are on it?